metomi/rose/date_cli.py

```diff
@@ -187,6 +185,10 @@ def main():
         file=sys.stderr
     )
 
+    if '--help' in sys.argv:
```

Comment:
`rose config-diff` on it's own returns `Usage: rose config-diff [options]` - fine as far as it goes, but shouldn't it return either the same as `--help` or a note to the effect of "see --help for details"?